recipes/conversation_view/feed_pill/feed_item_pill.stories.js

```diff
@@ -73,6 +73,31 @@ export const Default = {
   },
 };
 
+export const DefaultToggledVariant = {
+  render: DefaultTemplate,
+  parameters: { a11y: { disable: true } },
+  args: {
+    iconName: 'video',
+    title: 'This meeting has ended',
+    wrapperClass: 'd-w628',
+    ariaLabel: 'Click to expand',
+    toggleable: true,
+    defaultToggled: true,
+  },
+};
+
+export const DefaultToggledVariantNonToggleable = {
+  render: DefaultTemplate,
+  parameters: { a11y: { disable: true } },
+  args: {
+    iconName: 'video',
+    title: 'This meeting has ended',
+    wrapperClass: 'd-w628',
+    toggleable: false,
+    defaultToggled: true,
+  },
+};
```

Comment:
Please avoid duplicating code, if you're using the same args for multiple histories, extract them into a constant and just override the needed properties.

Cleanup the objects below if you can please, they're all using the same args. check other `.stories.js` files for guidance on the best way to do that.